uw-frame-components/portal/misc/partials/add-to-home.html

```diff
@@ -1,42 +1,31 @@
-<!-- ADD MORE TO HOME BUTTON -->
-<div ng-if='actionLinkUrl && !addToHome'>
-  <md-button class="md-primary link-div" ng-href="{{actionLinkUrl}}" hide-sm hide-xs>
-    <span><i class="fa fa-fw {{actionLinkIcon}}"></i> {{actionLinkText}}</span>
-  </md-button>
-  <md-menu-item hide-gt-sm>
-    <md-button class="md-primary link-div" ng-href="{{actionLinkUrl}}">
-      <span><i class="fa fa-fw {{actionLinkIcon}}"></i> {{actionLinkText}}</span>
-    </md-button>
-  </md-menu-item>
-</div>
-
 <!-- ADD -THIS- TO HOME BUTTON -->
-<div ng-controller="AddToHomeController" ng-if="addToHome">
+<div ng-controller="AddToHomeController">
   <!-- Button for medium+ screens -->
-  <md-button class="md-default link-div" aria-label="add to home" hide-xs hide-sm ng-hide='inHome' ng-click='addToHome()'>
+  <md-button class="md-default link-div" aria-label="add to home" hide-xs hide-sm ng-hide="inHome" ng-click="addToHome()">
     <span ng-if="!successfullyAdded || !addToHomeFailed">
-      <i class="fa fa-fw {{actionLinkIcon}}" ng-class="{ 'fa-spin' : savingAddToHome, 'fa-plus' : !actionLinkIcon}"></i> {{actionLinkText}}
+      <i class="fa fa-fw fa-plus" ng-class="{ 'fa-spin' : savingAddToHome }"></i> Add to home
     </span>
-    <span ng-show='successfullyAdded'>
-      <i class="fa fa-fw fa-check-square"></i> {{actionLinkText}} Saved
+    <span ng-show="successfullyAdded">
+      <i class="fa fa-fw fa-check-square"></i> Added to home
     </span>
-    <span ng-show='addToHomeFailed'>
-      <i class="fa fa-fw fa-exclamation-triangle"></i> {{actionLinkText}} Failed
+    <span ng-show="addToHomeFailed">
+      <i class="fa fa-fw fa-exclamation-triangle"></i> Add to home failed
     </span>
   </md-button>
 
   <!-- Button for small and xs screens -->
-  <md-menu-item hide-gt-sm>
-    <md-button class="md-default link-div" aria-label="add to home" ng-click="addToHome()" ng-hide="inHome">
+  <md-button class="md-default link-div" aria-label="add to home" hide-gt-sm ng-hide="inHome" ng-click="addToHome()">
     <span ng-if="!successfullyAdded || !addToHomeFailed">
-      <i class="fa fa-fw {{actionLinkIcon}}" ng-class="{ 'fa-spin' : savingAddToHome, 'fa-plus' : !actionLinkIcon}"></i> {{actionLinkText}}
+      <i class="fa fa-fw fa-plus" ng-class="{ 'fa-spin' : savingAddToHome }"></i>
     </span>
     <span ng-show="successfullyAdded">
-      <i class="fa fa-fw fa-check-square"></i> {{actionLinkText}} Saved
+      <i class="fa fa-fw fa-check-square"></i>
     </span>
     <span ng-show="addToHomeFailed">
-      <i class="fa fa-fw fa-exclamation-triangle"></i> {{actionLinkText}} Failed
+      <i class="fa fa-fw fa-exclamation-triangle"></i>
```

Comment:
Accessibility considerations on this visual-only indication that adding to home failed?